lambdas/patch-user/src/index.ts

```diff
@@ -0,0 +1,96 @@
+require('dotenv').config();
+
+import {UserData} from "../../../common/help-on-spot-models/src/models/RestModels";
+import {LambdaResponse, lambdaResponse} from "../../../common/help-on-spot-models/dist/utils/lambdaResponse";
+import {Address, Connection, Qualification, User, In} from "../../../common/help-on-spot-models/dist";
+import {Database} from "../../../common/help-on-spot-models/dist/utils/Database";
+
+
+export interface LambdaInputEvent {
```

Comment:
Move this to RestModels or somewhere else in commons?

Reply:
As discussed, not needed right now.